File: program_data/github_java_sort_function/6/1.java
private static void sortCount(int[] nums, int mask) {

        int[] counts = new int[10];
        int[] out = new int[nums.length];

        for (int i = 0; i < nums.length; i++) {
            counts[(nums[i] / mask) % 10]++;
        }

        for (int i = 1; i < 10; i++) {
            counts[i] += counts[i - 1];
        }

        for (int i = nums.length - 1; i >= 0; i--) {
            out[counts[(nums[i] / mask) % 10] - 1] = nums[i];
            counts[(nums[i] / mask) % 10]--;
        }

        System.arraycopy(out, 0, nums, 0, nums.length);

    }

    public static void radixSort(int[] nums) {

        int m = max(nums);

        for (int mask = 1; m / mask > 0; mask *= 10) {

            sortCount(nums, mask);

        }

    }

    

File: program_data/PKU_raw/82/1.c
main()
{
int n;
scanf("%d", &n);
int a[n], b[n]; int i;
for(i=0; i<n; i++)
   {
   scanf("%d %d", &a[i], &b[i]);
   }
int c[n]; int p=0;
for(i=0; i<n-1; i++)
   {
        if ((a[i]>=90&&a[i]<=140)&&(b[i]>=60&&b[i]<=90))
         {
         p=p+1;
         c[i]=0; 
         }
        else
         {
         c[i]=p; p=0;
         }
   }
if ((a[n-1]>=90&&a[n-1]<=140)&&(b[n-1]>=60&&b[n-1]<=90))
{ c[n-1]=p+1;}
else
{ c[n-1]=p;}
int Max=0;
for(i=0; i<n; i++)
   {
   if (Max<c[i])
   Max=c[i];
   }
printf("%d", Max);
}